torch/_dynamo/variables/builder.py

```diff
@@ -642,21 +642,43 @@ def wrap_module(self, value: torch.nn.Module):
             return self.tx.output.side_effects.track_object_existing(
                 self.source, value, result
             )
-        elif getattr(value, "_is_fsdp_managed_module", False) or issubclass(
+        elif issubclass(
```

Comment:
Something I wondered but did not check: Does this mean that DDP uses `UnspecializedNNModuleVariable`? Or when is this code path used?

Reply:
I haven't stepped through DDP in a while but iirc this only makes the DDP wrapper module itself an UnspecializedNNModule.  I can't remember why I did this.

But DDP doesn't have the 'poison' attribute like `_is_fsdp_managed_module`, so any modules inside the DDP wrapper should fall back to being regular NNModuleVariable.